source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -35,18 +65,24 @@ Controls Glossary
 
       Inputs are often represented by the variable :math:`\mathbf{u}`, a column vector with one entry per :term:`input` to the :term:`system`.
 
+   least-squares regression
+      A curve-fitting technique which picks a curve to minimizes the *square* of the error between the fitted curve, and the actual measured data. See `ordinary least-squares regression <https://en.wikipedia.org/wiki/Linear_regression>`__ on wikipedia for more info.
+
+   LQR
+      Linear-Quadratic Regulator - A feedback control scheme which seeks to operate a system in a "lowest cost" manner. It requires an accurate mathematical model of the system being controlled, and a set of equations describing what things are "costly". See `LQR <https://en.wikipedia.org/wiki/Linear%E2%80%93quadratic_regulator>`__ on wikipedia for more info.
```

Comment:
How about:

"Linear-Quadratic Regulator - A feedback control scheme which seeks to operate a system in an "optimal" or "lowest cost" manner, in the sense of minimizing the square of some "cost function" that represents a combination of system error and control effort. This requires an accurate mathematical model of the system being controlled, and function describing the "cost" of any given system state. See `LQR <https://en.wikipedia.org/wiki/Linear%E2%80%93quadratic_regulator>`__ on wikipedia for more info."

Reply:
much like, especially the "cost of any system state" part.